pkg/js/js.go

```diff
@@ -35,10 +42,25 @@ func ExecuteJavascript(file string, devMode bool, variables map[string]string) (
 	currentDirectory = filepath.Dir(file)
 
 	vm := otto.New()
+	l := loop.New(vm)
+
+	if err := timers.Define(vm, l); err != nil {
+		return nil, err
+	}
+	if err := promise.Define(vm, l); err != nil {
+		return nil, err
+	}
+	if err := fetch.Define(vm, l); err != nil {
+		return nil, err
+	}
+	if err := process.Define(vm, flag.Args()); err != nil {
```

Comment:
Why use github.com/jessevdk/go-flags when dnscontrol already processes flags using github.com/urfave/cli/v2 ?

Reply:
I wasn't aware of github.com/urfave/cli/v2 being used, this is copy pasted from the unit test of ottoext.

Upon closer inspection, all it does is expose environment variables and call arguments to the Javascript environment. We probably don't need that.

By the way, to my understanding it's not using github.com/jessevdk/go-flags. It's using Go's builtin flag package.

Reply:
This is removed for being unnecessary, and possibly causing credential leaks in some cases.